MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m

```diff
@@ -67,13 +71,12 @@ -(void) getSpotifyTracksArtists:(void (^)(NSDictionary *, NSError*))completion{
             // get dictionary of genres and artists based on top artists
             NSDictionary *artistDict = [self convertSpotifyArtists:artistArray];
             // get tracks data
-            NSLog(@"array: %@", artistArray);
             [self getSpotifyData:@"https://api.spotify.com/v1/me/top/tracks" completion:^(NSDictionary * tracksDict, NSError * error) {
                 if (!error){
                     NSArray *tracksArray = tracksDict[@"items"];
                     // get dictionary of genres, tracks, and artists based on top tracks
                     NSDictionary *tracksDict =[self convertSpotifyTracks:tracksArray];
-                    [artistDict[@"artists"] unionSet:tracksDict[@"artists"]];
+                     [artistDict[@"artists"] addObject:tracksDict[@"artists"]];
```

Comment:
Changed sets to array to store easily into the database

Reply:
Does this logic account for deduplication (in case there are duplicates from the add)?